benchexec/tablegenerator/template.html

```diff
@@ -1423,6 +1430,11 @@
   showScatterPlot();
 };
 
+function toggleFloatThead() {
+    $(".collapsable-header").toggle();
+    $("#dataTable").floatThead('reflow');
```

Comment:
Is this call to reflow really necessary? Doesn't it just slow down the whole toggling (it is quite slow)?